### ISSUE TITLE
Invalid quota entry acceptance tests fail intermittently

### ISSUE THREAD

**Issue by phil-davis** [owncloud/user_management#144]
e.g.
```
  Scenario Outline: change quota to an invalid value                                                          # /var/www/owncloud/apps/user_management/tests/acceptance/features/webUIManageQuota/manageUserQuota.feature:38
    When the administrator changes the quota of user "user1" to "<wished_quota>" using the webUI              # WebUIUsersContext::theAdministratorSetsTheQuotaOfUserUsingTheWebUI()
    Then a notification should be displayed on the webUI with the text 'Invalid quota value "<wished_quota>"' # WebUIGeneralContext::aNotificationShouldBeDisplayedOnTheWebUIWithTheText()
    And the quota of user "user1" should be set to "<wished_quota>" on the webUI                              # WebUIUsersContext::quotaOfUserShouldBeSetToOnTheWebUI()

    Examples:
      | wished_quota |
      | 30/40        |
        │ INFORMATION: timed out waiting for ajax calls to start
        Failed asserting that two strings are equal.
        --- Expected
        +++ Actual
        @@ @@
        -'Invalid quota value "30/40"'
        +''

--- Failed scenarios:

    /var/www/owncloud/apps/user_management/tests/acceptance/features/webUIManageQuota/manageUserQuota.feature:48

1 scenario (1 failed)
6 steps (4 passed, 1 failed, 1 skipped)
0m19.88s (20.43Mb)
webUI test rerun failed with exit status: 1
```
There seems to be some timing issue with actually seeing the notification message. The message comes for about 5 seconds - the test has to look "not early and not too late".

**Comment by individual-it** [comment]
reopening as the issue is not completely fixed by #148 
now we get 
```
        Exception: Page\OwncloudPage::waitTillXpathIsVisible xpath: //*[@id='notification'] timeout waiting for element to be visible in /var/www/owncloud/tests/acceptance/features/lib/OwncloudPage.php:170
```

**Comment by individual-it** [comment]
hopefully fixed by #152
